pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyGovernanceSql.java

```diff
@@ -33,8 +33,9 @@ long insertAgeOfMajorityRule(
             @Bind("order") int order);
 
     @GetGeneratedKeys
-    @SqlUpdate("insert into age_up_candidate (study_id, participant_user_id) values (:studyId, :userId)")
-    long insertAgeUpCandidate(@Bind("studyId") long studyId, @Bind("userId") long participantUserId);
+    @SqlUpdate("insert into age_up_candidate (study_id, participant_user_id, operator_user_id) values (:studyId, :userId, :operatorId)")
+    long insertAgeUpCandidate(@Bind("studyId") long studyId, @Bind("userId") long operatorUserId,
```

Comment:
names participantUserId and operatorUserId  seem to have swapped and doesn't reflect values they hold.
I believe should be userId, participantUserId and operatorUserId in the order

Reply:
Right. parameter names were mixed up - fixed